.env.dev

```diff
@@ -21,3 +21,7 @@ MEDIA_ROOT=/data/media
 POSTGRES_VERSION=11
 DEBUG=true
 TYPESENSE_API_KEY="apikey"
+
+# Mux settings
+MUX_TOKEN_ID=mux_token_id
+MUX_TOKEN_SECRET=mux_token_secret
```

Comment:
We will need the real versions to be in a SecretsManager Secret and exposed to the ECS tasks.